### PR TITLE
Update adminLTE to 2.4.18

### PR DESCRIPTION
**Pull request by PromoFaux**
**By submitting this pull request, I confirm the following:** 

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
**What does this PR aim to accomplish?:**

 - Change a few references per upgrade guide
 - define colors[] at the top of index.js as AdminLTE.options does not appear to exist. (there may be a better way of doing this)

Alternative to #1271, which does the same thing but modifies an older version of AdminLTE

Both paths achieve the same thing.

All credit due to @roccivic who took the time to triage the responsiveness issue

### CODE REVIEW

File: scripts/pi-hole/js/index.js [hunk 1]
```diff
@@ -10,6 +10,25 @@
...
 
+var colors = [
+  "#3c8dbc",
+  "#f56954",
+  "#00a65a",
+  "#00c0ef",
+  "#f39c12",
+  "#0073b7",
+  "#001F3F",
+  "#39CCCC",
```
Comment (XhmikosR) [inline scripts/pi-hole/js/index.js:21]: Can you lowercase all colors please?

**Comment by XhmikosR** [comment]
I wonder maybe we could update to the latest 2.x version while at it?

Assuming you tested the changes, then the remaining diff is this: https://github.com/ColorlibHQ/AdminLTE/compare/v2.4.12...v2.4.18

**Review by DL6ER — DISMISSED** [review]
Something is strange here, look at the colors in this graph:
Screenshot at 2020-05-12 16-57-13

So I cleared my cache and reloaded the page and everything *seemed* to be back to normal:
Screenshot at 2020-05-12 16-57-25

However, when reloading (with or without flushing the cache), I can reproduce both variants here in a seemingly nondeterministic fashion.

**edit**

This also applies to the other doughnut, however, not the bar graphs, it seems.

Screenshot at 2020-05-12 17-00-02
Screenshot at 2020-05-12 17-00-15

**edit 2**

I see what is causing the issue and will push a fix in a minute.

**Comment by PromoFaux** [comment]
> I wonder maybe we could update to the latest 2.x version while at it?

Huh, I could have sworn I grabbed the latest 2.4.x version, perhaps not!

> Something is strange here, look at the colors in this graph:

I took the colours from the documentation of AdminLTE <2.4 I may have got them wrong, of course!

**Comment by XhmikosR** [comment]
It seems another class is missing from `body` too `hold-transition` BTW.

**Comment by DL6ER** [comment]
I fixed the color issue in 74d6298. @XhmikosR You missed the Sign-Off in your commit. Do you want to signal your agreement in a comment? We can then set DCO to *pass* manually.

**Comment by XhmikosR** [comment]
@DL6ER I don't mind, you even squash it with the other patches which I personally prefer :)

**Comment by DL6ER** [comment]
@PromoFaux I see strange artifacts when scrolling with your last push. Something seems broken with 2.4.18.

```
git checkout 2314a60d
# Everything is fine

git checkout 9bf9e3b8
# I get lags when scrolling
```

I will try to record a GIF.

**Comment by PromoFaux** [comment]
That's odd.. seems OK to me when scrolling. Did you clear cache? I will await your gif :)

**Comment by DL6ER** [comment]
Yes, I cleared the cache and force-refreshed.
ezgif-1-99bbfc73bf03

See the odd dark blocks when scrolling. They are absent one commit earlier (again cleared cache before trying that).

I'm using:
Screenshot at 2020-05-12 17-29-08

**Comment by DL6ER** [comment]
Confirmed fixed after a Firefox update.

Screenshot at 2020-05-12 17-42-20

**Review by DL6ER — APPROVED** [review]

**Pull request merged**